### PR TITLE
Installed Ava and implemented first tests

### PR DESCRIPTION
**Pull request by rapala61**
#### What's this PR do?
Uses ava to implement the first tests in Gambit.

#### How should this be reviewed?
- Run `npm run test`. Tests should pass.

#### Any background context you want to provide?
Gambit is getting tested, so should you.

#### Relevant tickets
https://www.pivotaltracker.com/story/show/142542259

#### Checklist
- [x] Documentation added for new features/changed endpoints.

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -14,3 +14,4 @@
     strict:
       - 2
       - "global"
+    no-underscore-dangle: off
```
Comment (sergii-tkachenko) [inline .eslintrc]: Why?

File: lib/helpers.js [hunk 2]
```diff
@@ -26,6 +26,7 @@ module.exports.isValidZip = function (zip) {
 
 module.exports.isValidEmail = function (email) {
   /* eslint-disable max-len */
```
Comment (sergii-tkachenko) [inline lib/helpers.js]: Is it still necessary?

File: lib/helpers.js [hunk 3]
```diff
@@ -41,7 +42,7 @@ module.exports.containsNaughtyWords = function (stringToBeCensored) {
 
   // TODO: Rewrite to avoid using for loop.
   // @see https://github.com/airbnb/javascript#iterators-and-generators
-  for (let i = 0; i < naughtyWords.length; i++) {
+  for (let i = 0; i < naughtyWords.length; i += 1) {
```
Comment (sergii-tkachenko) [inline lib/helpers.js]: Replace ith forEach?

File: .eslintrc (hunk 1 above)
Comment (rapala61) [inline .eslintrc]: because of mongo and how it uses _id for its internal primary key

File: lib/helpers.js (hunk 2 above)
Comment (rapala61) [inline lib/helpers.js]: Nope. Removing.

**Pull request merged**